todoapp/projects/serializers.py

```
@@ -0,0 +1,29 @@
+from projects import models as projects_models
+from users import serializers as users_serializers
+
+
+class ProjectSerializer(serializers.ModelSerializer):
+    project_title = serializers.CharField(source="name")
+
+    class Meta:
+        model = projects_models.Project
+        fields = "__all__"
+
+    def __init__(self, *args, **kwargs):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)